zipkin_thrift_span_test.go

```diff
@@ -34,7 +34,9 @@ import (
 func TestThriftFirstInProcessSpan(t *testing.T) {
 	tracer, closer := NewTracer("DOOP",
 		NewConstSampler(true),
-		NewNullReporter())
+		NewNullReporter(),
+		TracerOptions.Gen128Bit(true),
```

Comment:
Please use a separate test for 128bit. Conditions to check: high id is nil or some value

Reply:
Done. PTAL.